app/controllers/orga/projects_controller.rb

```diff
@@ -14,6 +14,15 @@ def accept
     redirect_to [:orga, :projects]
   end
 
+  def pending
+    if @project.pending!
+      flash[:notice] = "Project review still in progress!"
+    else
+      flash[:alert]  = "There has been an error accepting this project."
```

Comment:
This error message looks like copy/paste error. It should be something like "There has been an error marking this project as pending."